app/src/main/java/helium314/keyboard/latin/settings/CorrectionSettingsFragment.java

```diff
@@ -76,6 +76,7 @@ private void refreshEnabledSettings() {
         setPreferenceVisible(Settings.PREF_MORE_AUTO_CORRECTION, Settings.readAutoCorrectEnabled(getSharedPreferences()));
         setPreferenceVisible(Settings.PREF_ADD_TO_PERSONAL_DICTIONARY, getSharedPreferences().getBoolean(Settings.PREF_KEY_USE_PERSONALIZED_DICTS, true));
         setPreferenceVisible(Settings.PREF_ALWAYS_SHOW_SUGGESTIONS, getSharedPreferences().getBoolean(Settings.PREF_SHOW_SUGGESTIONS, true));
+        setPreferenceVisible(Settings.PREF_AUTO_HIDE_TOOLBAR, getSharedPreferences().getBoolean(Settings.PREF_SHOW_SUGGESTIONS, true));
```

Comment:
You can remove this. I plan to move the toolbar settings to a separate section, now that there are more and more toolbar-specific settings available.